### PR TITLE
added the missing components dependencies to pseudo module standard components

### PR DESCRIPTION
**Pull request by ClaudiaPHI**
added the missing components dependencies and removed the unnecessary
ones from desktop ui

### CODE REVIEW

File: desktop/ui/pom.xml [hunk 1]
```diff
@@ -72,28 +67,6 @@
 			<artifactId>DataCleaner-env-berkeleydb</artifactId>
 			<version>${project.version}</version>
 		</dependency>
-
-		<!-- DataCleaner components -->
-		<dependency>
-			<groupId>org.eobjects.datacleaner</groupId>
-			<artifactId>DataCleaner-standard-components</artifactId>
```
Comment (kaspersorensen) [inline desktop/ui/pom.xml]: removing the standard-components?

File: standard-components/pom.xml [hunk 2]
```diff
@@ -80,27 +90,27 @@
 		</dependency>
 		<dependency>
 			<groupId>org.eobjects.datacleaner</groupId>
-			<artifactId>DataCleaner-json</artifactId>
+			<artifactId>DataCleaner-sample-extension</artifactId>
```
Comment (kaspersorensen) [inline standard-components/pom.xml]: We don't want the sample-extension in here. It's only a coding example, not something to actually use!

**Comment by kaspersorensen** [comment]
Hmm the diff is pretty impossible to read because it seems to have shifted a lot of the dependencies down or up. I cannot figure out fully what changed.

File: desktop/ui/pom.xml (hunk 1 above)
Comment (ClaudiaPHI) [inline desktop/ui/pom.xml]: it is included in xml-config dependency

**Comment by ClaudiaPHI** [comment]
I added the components you asked me about in the email.

**Comment by kaspersorensen** [comment]
I only asked why, didn't say you had to :) Anyways I think this also adds sample-extension? And the diff makes it hard for me to see other effects.

File: desktop/ui/pom.xml (hunk 1 above)
Comment (kaspersorensen) [inline desktop/ui/pom.xml]: OK but I think it's best to explicitly name it here. The transitive dependency of xml-config is more a coincidence than design, whereas "ui" depending on "api" is certainly a strong contract.

**Comment by ClaudiaPHI** [comment]
I also changed the order in order to match the order in the one in project tree.  If this is not necessary, we can close the branch.

File: desktop/ui/pom.xml (hunk 1 above)
Comment (ClaudiaPHI) [inline desktop/ui/pom.xml]: Ok. it seems the original thing is better.

**Comment by ClaudiaPHI** [comment]
If you do not mind I will close this pull request without merging.

**Comment by kaspersorensen** [comment]
Ok with me :-)

**Pull request closed without merging**